scipy/stats/tests/test_qmc.py

```diff
@@ -570,6 +574,16 @@ def test_warning(self):
             seed = np.random.RandomState(12345)
             qmc.LatinHypercube(1, seed=seed, optimization="toto")
 
+        with pytest.raises(ValueError, match=r"not a prime number"):
+            qmc.LatinHypercube(d=2, orthogonal_array_p=4, seed=seed)
+
+        with pytest.raises(ValueError, match=r"d is too large"):
+            qmc.LatinHypercube(d=5, orthogonal_array_p=3, seed=seed)
+
+        with pytest.raises(ValueError, match=r"n is not equal"):
+            engine = qmc.LatinHypercube(d=4, orthogonal_array_p=3, seed=seed)
+            engine.random(10)
+
```

Comment:
We need a test that confirms that the Latin hypercube is orthogonal.

Reply:
I added such test thanks to a suggestion from @kstoneriv3.